<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>gro : the cell programming language</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionDump"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStartStop"/>
    <addaction name="actionStep"/>
    <addaction name="actionMoreCells"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionReload"/>
   <addaction name="separator"/>
   <addaction name="actionStartStop"/>
   <addaction name="actionStep"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/start.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start or stop gro </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a .gro file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/reload.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload current .gro file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/step.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Simulate for one timestep</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Online Documentation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Screenshot...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionMoreCells">
   <property name="text">
    <string>Increase Population Limit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/zoom_in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/zoom_out.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionDump">
   <property name="text">
    <string>Save Colony Information...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionStartStop</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>startStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStep</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>snapshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMoreCells</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>moreCells()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_In</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>zoom_in()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_Out</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>zoom_out()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset_Zoom</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>reset_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDump</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>dump()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStop()</slot>
  <slot>open()</slot>
  <slot>reload()</slot>
  <slot>step()</slot>
  <slot>slot2()</slot>
  <slot>help()</slot>
  <slot>about()</slot>
  <slot>snapshot()</slot>
  <slot>moreCells()</slot>
  <slot>zoom_in()</slot>
  <slot>zoom_out()</slot>
  <slot>reset_zoom()</slot>
  <slot>dump()</slot>
 </slots>
</ui>
